# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1502381033 14400
#      Thu Aug 10 12:03:53 2017 -0400
# Node ID e660b93d51dc1635802cd396c1ade7cfe527ae28
# Parent  56b7352fa4565b4f5cc9ceaff0429ee7a1aea0a3
* classdef.tst: New test (bug #51715).

diff --git a/test/classdef/classdef.tst b/test/classdef/classdef.tst
--- a/test/classdef/classdef.tst
+++ b/test/classdef/classdef.tst
@@ -70,8 +70,15 @@
 %!test
 %! obj = foo_method_changes_property_size (3);
 %! obj = obj.move_element_to_end (2);
 %! assert (obj.element, [1 3 2]);
 
 %!error <parse error> plist_t1
 %!assert (class (plist_t2), "plist_t2")
 %!assert (class (plist_t3), "plist_t3")
+
+## Do not change this to "containers.Map()".  This test is intended to
+## ensure that calling a function in a +package directory will work
+## properly.
+%!test <*51715>
+%! x = containers.Map;
+%! assert (isobject (x));
